docs/source/redis.rst

```diff
@@ -0,0 +1,62 @@
+.. toctree::
```

Comment:
I've since figured out how to fix this:
```
html_sidebars = {"**": ["globaltoc.html", "relations.html", "sourcelink.html", "searchbox.html"]}
```
in the conf.py, and you should be able to omit this toctree directive. though double check what happens on pages that currently have it still to make sure it doesn't look stupid, if it does please remove it from everything but the index.rst